src/components/ListItem.jsx

```diff
@@ -17,7 +17,7 @@ const calculateIsPurchased = (dateLastPurchased) => {
 	return currentDate < oneDayLater;
 };
 
-export function ListItem({ item, listPath }) {
+export function ListItem({ item, listPath, className }) {
```

Comment:
I see what you're trying to do here! Could I suggest a cleaner approach? Instead of passing a `className`, we should pass an `urgencyStatus` prop. This way, the `className` can be determined internally based on the `urgencyStatus`, keeping the styling logic encapsulated within the `ListItem` component. This approach is more semantic, makes the component more reusable, and keeps the parent component simpler.

Reply:
Thank you! I am adding this as a change to useUrgency and use-urgency branch